recipes/selenium/Dockerfile

```diff
@@ -30,7 +30,6 @@ RUN sudo apt-get update -qqy && \
   x11vnc \
   xvfb \
   subversion \
-  openjdk-8-jdk-headless \
```

Comment:
that image is `FROM eclipse/stack-base:ubuntu` so it already have jdk from parent it was just redundant